hw/drivers/flash/at45db/src/at45db.c

```diff
@@ -446,7 +446,10 @@ at45db_init(const struct hal_flash *hal_flash_dev)
     }
 
     hal_spi_set_txrx_cb(dev->spi_num, NULL, NULL);
-    hal_spi_enable(dev->spi_num);
+    rc = hal_spi_enable(dev->spi_num);
```

Comment:
i would just return rc all the time since there is nothing to do after if (rc) statement

Reply:
ok, done